net/udpspeeder/Makefile

```diff
@@ -0,0 +1,63 @@
+#
+# Copyright (c) 2017 Yu Wang <wangyucn@gmail.com>
+# Copyright (c) 2020 Nishant Sharma <nishant@hopbox.in>
+#
+# This is free software, licensed under MIT
+#
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=udpspeeder
+PKG_VERSION:=20200818.0
```

Comment:
There's a newer version available.

Reply:
Thanks for your review. Updated to the newer version.